go/teams/handler.go

```diff
@@ -445,6 +448,22 @@ func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase
 			return removeInviteID(ctx, team, invite.Id)
 		}
 
+		var convID chat1.ConversationID
+		var iteamName string
+		if team.IsImplicit() {
+			iteamName, err = team.ImplicitTeamDisplayNameString(ctx)
+			if err != nil {
+				return err
+			}
+			uid := gregor1.UID(g.ActiveDevice.UID().ToBytes())
+			conv, err := g.ChatHelper.NewConversation(ctx, uid, iteamName, nil,
+				chat1.TopicType_CHAT, chat1.ConversationMembersType_IMPTEAMNATIVE, keybase1.TLFVisibility_PRIVATE)
+			if err != nil {
+				return err
+			}
+			convID = conv.GetConvID()
+		}
+
```

Comment:
let's factor this out to some `struct ImpteamConversationRef { convID, impteamName }` that lives in another file (`go/teams/chatmessaging.go` is probably cool for now), this function does too many things already.

so we can have this almost 1-liner here:
```
// Save conversation reference now before we resolve the team invite,
// because if there are already implicit teams with same display name as our
// "target" name, we won't be able to find the conversation afterwards.
impteamConvRef := GetImpteamConversationRef(ctx, g, team)
```
and then pass it to `SendChatSBSResolutionMessage`.